keyboards/alps64/alps64.h

```diff
@@ -106,21 +106,4 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
     { K70,   K71,   K72,   K73, K74, K75, K76,   K77 } \
 }
 
-#define LAYOUT_kc( \
```

Comment:
I am not in favor of removing the LAYOUT_kc macro as people do find this useful. 

It's fine that we made LAYOUT_all the macro used by the default keymap, but we should keep LAYOUT_kc